config/config.go

```diff
@@ -704,7 +704,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.sonobi.endpoint", "https://apex.go.sonobi.com/prebid?partnerid=71d9d3d8af")
 	v.SetDefault("adapters.sovrn.endpoint", "http://ap.lijit.com/rtb/bid?src=prebid_server")
 	v.SetDefault("adapters.synacormedia.endpoint", "http://{{.Host}}.technoratimedia.com/openrtb/bids/{{.Host}}")
-	v.SetDefault("adapters.tappx.endpoint", "https://{{.Host}}")
+	v.SetDefault("adapters.tappx.endpoint", "http://{{.Host}}")
```

Comment:
We are no longer allowing any host to be defined. See: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#bidder-parameters

Are you currently using this for testing, geo balancing, or some other purpose?

Reply:
We using now because for privacy and each partner is different.

Reply:
Do the partners have fully different domains or are they subdomains? We'll need to fix this such that the entire domain is not specified by the request.

Reply:
Hi! Sorry for delay, I don't now the answer and ask to my boss. He explain to me that we are using fully domains and we prefer to still using {.host} if is posible. Still working or is something that doesn't work?
Thanks for your patience!

Reply:
Allowing the publisher to specify the full domian/host through the request exposes the PBS host to unfairness and even possible danger. For example, the Tappx adapter may consume more than its fair share of outgoing connections (we're currently working on solutions to mitigate that issue) and unwittingly being used maliciously such as in a DDOS attack.

We no longer allow this behavior in adapters. I recognize your adapter predates our formal stance. If you could please initiate a discussion with me over email (skay@prebid.org) we can move forward with both merging in this PR and working together on a long term solution.

Reply:
Email received. Thank you. We'll continue this discussion offline.